<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComposantPrincipal</class>
 <widget class="QMainWindow" name="ComposantPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>481</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>641</width>
    <height>481</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>641</width>
    <height>481</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestionnaire de Mots de passe</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="pageConnexion">
     <widget class="QLabel" name="titrePageConnexion">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Connexion</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="descriptionPageConnexion">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>130</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Veuillez vous identifier sur le système</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="identifiantPageConnexion">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>160</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>50</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Identifiant</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mdpPageConnexion">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>190</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Mot de passe</string>
      </property>
     </widget>
     <widget class="QPushButton" name="connexionPageConnexion">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>220</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Connexion</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageListeMDP">
     <widget class="QLabel" name="titrePageListeMDP">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Vos mots de passe</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QListWidget" name="listeMotsDePasse">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>251</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelNomServeur">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>110</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelHoteServeur">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>140</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelIdentifiant">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>200</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelNom">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>170</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelInformations">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Mot de passe sélectionné</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label1">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>110</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom du serveur :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>140</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Hôte du serveur :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>170</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom du mot de passe :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>200</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifiant :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label5">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>230</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Mot de passe :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="motDePasse">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>230</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="afficherMasquer">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>260</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="actualiser">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>420</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Actualiser</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageConfiguration">
     <widget class="QLabel" name="titrePageConfiguration">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Configuration initiale</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="descriptionPageConfiguration">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Aucune configuration valable n'a été trouvée.&lt;/p&gt;&lt;p&gt;Veuillez entrer l'adresse ip ou le nom d'hôte du serveur ainsi que son port d'écoute :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelServeurPageConfiguration">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>46</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Serveur :</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPortPageConfiguration">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Port :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="serveurPageConfiguration">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="portPageConfiguration">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
     </widget>
     <widget class="QPushButton" name="enregistrerPageConfiguration">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>190</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Enregistrer et connecter</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionDeconnexion"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuAdministration">
    <property name="title">
     <string>Administration</string>
    </property>
    <addaction name="actionGestionUtilisateurs"/>
    <addaction name="actionGestionGroupes"/>
    <addaction name="actionGestionServeurs"/>
    <addaction name="actionGestionMDP"/>
    <addaction name="separator"/>
    <addaction name="actionSauvegarder"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuAdministration"/>
  </widget>
  <action name="actionDeconnexion">
   <property name="text">
    <string>Déconnexion</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionGestionUtilisateurs">
   <property name="text">
    <string>Gestion utilisateurs</string>
   </property>
  </action>
  <action name="actionGestionGroupes">
   <property name="text">
    <string>Gestion groupes</string>
   </property>
  </action>
  <action name="actionGestionServeurs">
   <property name="text">
    <string>Gestion serveurs</string>
   </property>
  </action>
  <action name="actionGestionMDP">
   <property name="text">
    <string>Gestion mots de passe</string>
   </property>
  </action>
  <action name="actionSauvegarder">
   <property name="text">
    <string>Sauvegarder le serveur</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
